website/docs/reference/glossary.md

```diff
@@ -0,0 +1,140 @@
+---
+sidebar_position: 2
+---
+
+# Glossary
+
+## List of FerretDB terminologies
+
+*This section contains a list of common terminologies related to FerretDB*.
+
+### A
+
+#### aggregation
+
+A group of operations used to reduce or merge large data sets.
+See [list of supported aggregation operations and commands here](./supported_commands.md#aggregation-pipelines).
+
+#### aggregation pipeline
+
+A set of operators that lets you perform complex operations that aggregate and summarize values.
+See [list of supported aggregation pipeline operators](./supported_commands.md#aggregation-pipeline-operators) here.
+
+---
+
+### B
+
+#### Beacon
+
+The telemetry service of FerretDB.
+See [telemetry](../telemetry.md) for more details.
+
+#### BSON
+
+BSON is a serialized binary file format for storing JSON-like documents.
+
+#### BSON types
+
+The list of types that the BSON format supports.
+BSON offers support for additional data types compared to JSON, such as `timestamp`, `date`, `ObjectId`, and `binary`.
+
+---
+
+### C
+
+#### collection
+
+A group of documents in a non-relational database.
+It is comparable to a table in a relational database.
+
+#### CRUD
+
+The four basic operations of a database: Create, Read, Update, and Delete.
+See [Basic FerretDB CRUD operations here](../basic_operations/index.md).
+
+---
+
+### D
+
+#### database
+
+An organized repository for collections containing its own sets of documents, and data.
+
+#### database command
+
+The set of commands in FerretDB.
+For more information, see [supported commands](./supported_commands.md) for more details.
+
+#### document
+
+A record in a collection that comprises key-value pairs.
+See [Documents](../understanding_ferretdb.md#documents) for more.
+
+#### dot notation
+
+Dot notation is used to reference or access the elements in an array or in an embedded document.
+See [dot notation](../understanding_ferretdb.md#dot-notation) for more details.
+
+---
+
+### F
+
+#### field
+
+Similar to columns in a relational database.
+They are represented as field name-value pairs and describe the kind of data in a document.
+
+---
+
+### I
+
+#### index
+
+A data structure used for identifying and querying records in a collection.
+It helps to limit the number of documents to search through or inspect in a collection.
+Examples include unique `_id` index, user-defined index, hashed index, and partial index.
+
+---
+
+### J
+
+#### JSON
+
+An acronym for JavaScript Object Notation.
+It is a structured data format with human-readable text to store data objects composed of attribute-value pairs.
+
+#### JSONB
+
+JSONB is a data type of PostgreSQL that stores JSON data as a decomposed binary format.
+
+#### ObjectId
+
+A defining 12-byte type that ensures singularity and uniques within a collection and are used to represent the default values for the `_id` fields.
+
+#### operator
```

Comment:
How do we call things like `insert`, `update`, `find`, `count` etc? Methods? Operations? Should we add a line about those things to our glossary?

Reply:
Good question. They should be operations, and in some instances,  commands. I'm no longer sure if method is appropriate. Let me know what you think.

Reply:
I'm not sure either :) I think, currently, quite often, we use "methods" or "commands" as synonyms, and I think this is one of the reasons why we want to have a glossary - to know how to call them correctly. I see that in the code we call things like "insert", "find" etc commands - https://github.com/FerretDB/FerretDB/blob/7c94741baebed1cb24040a9a712353a1f174a394/internal/handlers/common/msg_listcommands.go#L42. So, maybe "command" is a good word.

Reply:
Let's use "commands" for them going forward.